docs/src/main/paradox/scala/http/routing-dsl/testkit.md

```diff
@@ -110,6 +110,23 @@ This is what the full working test looks like:
 @@snip [TestKitFragmentSpec.scala](../../../../../test/scala/docs/http/scaladsl/server/TestKitFragmentSpec.scala) { #source-quote }
 
 
+## Accounting for Slow Test Systems
+
+Sometimes, the timeout defined for a route is finely tuned for our production environment, but our Continuous
```

Comment:
Regarding Java side, the default timeout is not yet using dilated, this is why I didn't add it there.